### ISSUE TITLE
Unable to install due to compilation errors

### ISSUE THREAD

**Issue by adamsoutar** [ReactUnity/core#13]
Hi there,

This looks to be a very exciting project, solving my number one gripe with Unity UI, and I'm very excited to use it.

Unfortunately, when I try to install it, I'm getting compilation errors in Unity.

I tried at first with Unity 2021.1.12f1, which gives a tonne of them:
(Same result with openupm or the Package Manager window)

<img width="1584" alt="image" src="https://user-images.githubusercontent.com/26578100/123528002-00d46200-d6dc-11eb-87fa-a2e75afec982.png">

Then with Unity 2020.3.12f1, which gives me just one.

<img width="1116" alt="image" src="https://user-images.githubusercontent.com/26578100/123528011-1d709a00-d6dc-11eb-93c9-e6490a011326.png">

Looking into the code just quickly, I thought it looked like there might be a typo on line 81 of ReactWindow.cs
where it might be supposed to say

```
protected JavascriptEngineType EngineType
```

instead of

```
protected JavascriptEngineType JavascriptEngineType
```

But I'm not sure. (Unity won't let me edit the code of the package on my local machine - I assume it checks with hashes etc.)

Have you seen either of these situations before & do you know how I might solve them?

Thanks again for working on such an interesting project.

**Comment by KurtGokhan** [comment]
The first error is caused because TextMeshPro is not installed. You can install it from package manager (version > 3 required). 

The second error was caused by a bug as you said. Should be fixed with latest updates (0.0.39). 

Sorry for the inconvenience 😔

**Comment by KurtGokhan** [comment]
Waiting for game-ci/unity-actions/issues#109 to be fixed to close this. Normally the Github Action should fail when this kind of error happens.
